vllm/core/block_manager_v1.py

```diff
@@ -473,11 +473,12 @@ def can_swap_in(self,
 
     def swap_in(self,
                 seq_group: SequenceGroup,
-                num_lookahead_slots: int = 0) -> Dict[int, int]:
+                num_lookahead_slots: int = 0) -> List[Tuple[int, int]]:
         assert (num_lookahead_slots == 0
                 ), "BlockSpaceManagerV1 does not support lookahead allocation"
 
         # CPU block -> GPU block.
+        # dict is efficient in lookup `if cpu_block in mapping`
```

Comment:
is it a better approach if we do `list(mapping.items())` in the worker layer?

Reply:
IMO the mapping semantics are better represented by dict[int, int], which is why I ask

Reply:
@cadedaniel This is basically for performance. List append/extend is faster than Dict insert/update.